h2o-py/tests/testdir_generic_model/pyunit_generic_model_mojo_gbm.py

```diff
@@ -46,6 +51,8 @@ def test(x, y, output_test, strip_part, algo_name, generic_algo_name):
     assert len(generic_mojo_model_from_file._model_json["output"]["variable_importances"]._cell_values) > 0
     assert generic_mojo_model_from_file._model_json["output"]["model_summary"] is not None
     assert len(generic_mojo_model_from_file._model_json["output"]["model_summary"]._cell_values) > 0
+    assert generic_mojo_model_from_file._parms is not None
+    assert len(generic_mojo_model_from_file._parms._cell_values) > 0
```

Comment:
When I try to inspect the params inside `generic_mojo_model_from_file.parms`

I get

```
<class 'dict'>: {'model_id': {'__meta': {'schema_version': 3, 'schema_name': 'ModelParameterSchemaV3', 'schema_type': 'Iced'}, 'name': 'model_id', 'label': 'model_id', 'help': 'Destination id for this model; auto-generated if not specified.', 'required': False, 'type': 'Key<Model>', 'default_value': None, 'actual_value': {'__meta': {'schema_version': 3, 'schema_name': 'ModelKeyV3', 'schema_type': 'Key<Model>'}, 'name': 'Generic_model_python_1566302911212_23', 'type': 'Key<Model>', 'URL': '/3/Models/Generic_model_python_1566302911212_23'}, 'level': 'critical', 'values': [], 'is_member_of_frames': [], 'is_mutually_exclusive_with': [], 'gridable': False}, 'model_key': {'__meta': {'schema_version': 3, 'schema_name': 'ModelParameterSchemaV3', 'schema_type': 'Iced'}, 'name': 'model_key', 'label': 'model_key', 'help': 'Key to the self-contained model archive already uploaded to H2O.', 'required': False, 'type': 'Key<Frame>', 'default_value': None, 'actual_value': {'__meta': {'schema_version': 3, 'schema_name': 'FrameKeyV3', 'schema_type': 'Key<Frame>'}, 'name': 'nfs://tmp/tmpgy08jywf/GBM_model_python_1566302911212_1.zip', 'type': 'Key<Frame>', 'URL': '/3/Frames/nfs://tmp/tmpgy08jywf/GBM_model_python_1566302911212_1.zip'}, 'level': 'critical', 'values': [], 'is_member_of_frames': [], 'is_mutually_exclusive_with': [], 'gridable': False}, 'path': {'__meta': {'schema_version': 3, 'schema_name': 'ModelParameterSchemaV3', 'schema_type': 'Iced'}, 'name': 'path', 'label': 'path', 'help': 'Path to file with self-contained model archive.', 'required': False, 'type': 'string', 'default_value': None, 'actual_value': '/tmp/tmpgy08jywf/GBM_model_python_1566302911212_1.zip', 'level': 'critical', 'values': [], 'is_member_of_frames': [], 'is_mutually_exclusive_with': [], 'gridable': False}}
```

**Question**: Is that all that is there ?

I've downloaded the MOJO created in this test 
[tmpmojo.zip](https://github.com/h2oai/h2o-3/files/3520539/tmpmojo.zip) - 
 and I can see much more in the parameters section of modelDetails than is present in the model._parms.